python/ray/services.py

```diff
@@ -577,13 +584,17 @@ def start_reaper():
     Returns:
         ProcessInfo for the process that was started.
     """
+    if ray.utils.detect_fate_sharing_support():
```

Comment:
It's probably more clear to have this check guarding whether or not we call `start_reaper` instead of inside the function

Reply:
Yeah sure, I can change that.